docs/README.md

```diff
@@ -1876,36 +1868,38 @@ Where:
 
-In the following examples we use ["wscat"](https://github.com/websockets/wscat) to connect to the web socket and send a subscription
-to the read model `CartReadModel` with ID "demo"
-
-Remember that in case you want to subscribe to a read model that is restricted to a specific set of roles, you must send, 
-in the *connection operation*, the **access token** in the **"Authorization"** header: 
-*"Authorization: Bearer &lt;token retrieved upon sign-in&gt;"*
-
-> Connecting to the web socket:
+In the following examples we use ["wscat"](https://github.com/websockets/wscat) to connect to the web socket. After that, we send the required messages to conform the ["GraphQL over WebSocket" protocol](#the-graphql-over-websocket-protocol), including the subscription operation
+to the read model `CartReadModel` with ID "demo".
 
+1. Connect to the web socket:
 ```sh
  wscat -c <websocketURL> -s graphql-ws
 ```
+> **Note:** You should specify the `graphql-ws` subprotocol when connecting with your client via the `Sec-WebSocket-Protocol` header (in this case, `wscat` does that when you use the `-s` option).
 
-**Note:** You should specify the `graphql-ws` subprotocol to your clients via the `Sec-WebSocket-Protocol` header.
-
-> Sending a message with the subscription
-
+Now we can start sending messages just by writing them and hitting "Enter".
+2. Initiate the protocol connection :
+```json
+{ "type": "connection_init" }
+```
+In case you want to authorize the connection, you need to send the authorization token in the `payload.Authorization` field:
+ ```json
+ { "type": "connection_init", "payload": { "Authorization": "<your token>" } }
+ ```
+3. Send a message with the subscription. We need to provide an ID for the operation. When the server sends us data back, it will include this same ID so that we know which subscription the received data belongs to (again, this is just for learning, [GraphQL clients](#using-apollo-client) manages this for you)
 ```json
-{"query": "subscription { CartReadModel(id:\"demo\") { id items } }" }
+{ "id": "1", "type": "start", "payload": { "query": "subscription { CartReadModel(id:\"demo\") { id items } }" } }
 ```
 
 After a successful subscription, you won't receive anything in return. Now, every time the read model you subscribed to
 is modified, a new incoming message will appear in the socket with the updated version of the read model. This message
 will have exactly the same format as if you were done a query with the same parameters.
 
-Following the previous example, we now send a command (using a "mutation" operation) that adds
+Following with the previous example, we now send a command (using a "mutation" operation) that adds
 a new item with sku "ABC_02" to the `CartReadModel`. After it has been added, we receive the updated version of the read model through the
 socket.
 
-> Send the Command
+1. Send the following command:
```

Comment:
What do you think about to specify here that we are creating a totally different call using the rest API to be more clear about what we are doing?

Reply:
Make sense! Done here: b7df753be5bb1e66410e2b58ddd1638ccfb27c74

What do you think?